src/components/MapLibreMap/lib/MapLibreGlWrapper.ts

```diff
@@ -12,6 +12,16 @@ import {
 	StyleImageMetadata,
 } from 'maplibre-gl';
 import { Map as MapType, Style } from 'maplibre-gl';
+import maplibregl from 'maplibre-gl';
+
```

Comment:
I think we must move this to its own hook. Let's call it `useSetRtlTextPlugin` (what a beautiful name :)).
This way we can use it optionally when we actually need it.

Please take a look at the useAddProtocol hook which might have a similar implementation
https://github.com/mapcomponents/react-map-components-maplibre/blob/main/src/hooks/useAddProtocol/useAddProtocol.tsx